unlock-app/src/config/app.ts

```diff
@@ -45,10 +45,11 @@ const app = getAppConfig(env)
 
 export const config = {
   env,
-  wedlocksPublicKey: process.env.NEXT_PUBLIC_BASE64_WEDLOCKS_PUBLIC_KEY!,
+  wedlocksPublicKey:
+    'LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0NCk1JSUJJakFOQmdrcWhraUc5dzBCQVFFRkFBT0NBUThBTUlJQkNnS0NBUUVBbzdTYXhDZDg3YnQ0SFZ4anhXbkkNClFOTGFhZVdqb1ptbFNxcGhkL1JCbDNzNGQrcENnZHl3YUJBdGRqVENMQXBzakN6SzFqZWZLVWZmbXhqbW15NGUNCnFHRHN2ekZBazRUS1ZrM0g4ZTJrYnJpNHdoZjNaU1V4d2gyL2c2WWgrRzFuK1F2cWJ0ZGwvOVUvcnJ5SmR5NFENCmt6K2tJMzBsRk9aNHJCMHJ5bldvdDZuZGtISUVlMDFhQThzYWpSb1ZrZitDb1RFZ1hWNlowd2gzSlRNc01FUEENCk8rc2FGYkVaZlI4Y1lMNEVVanB1Ty9WUXZyZk1nVDRiQUxLQXVrV1hweGZWZDNWTVlNallQeU52KzUwRTFRR2UNClZYN0xtYUR6ZmhuQlhTbG4zU01mYncvTWl2cExoR3RlQ0NIN0JYaU8zb1hQUFpVd3ZIT3BzTm1OS2F6dEx4OG4NCmF3SURBUUFCDQotLS0tLUVORCBQVUJMSUMgS0VZLS0tLS0NCg==',
   stripeApiKey:
     process.env.NEXT_PUBLIC_STRIPE_KEY || 'pk_test_BHXKmScocCfrQ1oW8HTmnVrB',
-  ethPassApiKey: process.env.NEXT_PUBLIC_ETHPASS_KEY!,
+  ethPassApiKey: 'pk_live_Th4KfDH9DZw0cuHcWzyUFBgAeHEl0MoK',
```

Comment:
I know it's easily available from public site but pushing them to github makes it liable to get scrapped by some bot who is looking to spam + github automatically disables some keys it finds via secret scanning partner program.

Reply:
Ha! I can move that to 1passwird if that's an issue .. I'll do it